### ISSUE TITLE
Source Maps not being correctly built

### ISSUE THREAD

**Issue by cavalloTNguyen** [smapiot/piral-cli-vite#3]
# Bug Report

## Prerequisites

- [x] Can you reproduce the problem in a [MWE](https://en.wikipedia.org/wiki/Minimal_working_example)?
- [x] Are you running the latest version?
- [x] Did you perform a search in the issues?

## Environment Details and Version

[Package Version, OS, Browser, ...]
`piral`: 1.3.3
`piral-cli`: 1.3.3
`piral-cli-vite`: latest (1.0.0)
OS: macOS 13,
Browser: Chrome 118.0.5993.117  ARM64

## Description

Sourcemaps are not being properly generated when using vite as a pilet bundler. 

## Steps to Reproduce

1. Spin up an App Shell following instructions here: https://docs.piral.io/guidelines/tutorials/02-getting-started
2. Build the App Shell Emulator
3. Spin up a Pilet using Vite and the Bundler 
4. Run the pilet

## Expected behavior

Message regarding the pilet plugin not generating a sourceamp should not appear in terminal
Output should have sourcemaps (be searchable in Chrome Dev Tools, debuggable in VSCode) 

## Actual behavior

The following message shows in terminal when running the pilet:
```
(pilet plugin) Sourcemap is likely to be incorrect: a plugin (pilet) was used to transform files, but didn't generate a sourcemap for the transformation. Consult the plugin documentation for help
```

Sourcemaps are not generated, unable to set breakpoints in VSCode after launching it against the app in development, unable to view the mapped files in Chrome Dev Tools

**Comment by FlorianRappl** [comment]
I'm looking into it. What I can say right now is that its reproducible and that the `vite-plugin-pilet` is causing the issue.

**Comment by FlorianRappl** [comment]
Fixed with 1.0.1 of `piral-cli-vite`.
